### PR TITLE
Feat/new landing page

### PR DESCRIPTION
**Pull request by Emilsone**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -18,15 +18,16 @@
...
 					<span class="ml-3 text-xl">Reduce</span>
 				</a>
 				<nav class="md:ml-auto flex flex-wrap items-center text-base justify-center">
-					<a class="mr-5 hover:text-gray-900">About</a>
-					<a class="mr-5 hover:text-gray-900">Solution</a>
-					<a class="mr-5 hover:text-gray-900">Login</a>
+					<a class="mr-5 hover:text-gray-900">ABOUT</a>
+					<a class="mr-5 hover:text-gray-900">LOGIN</a>
+					<a class="mr-5 hover:text-gray-900">REGISTER</a>
+					<a class="mr-5 hover:text-gray-900">DASHBOARD</a>
```
Comment (micaiah-effiong) [inline public/index.html]: What is the dashboard link for...?

File: public/index.html [hunk 2]
```diff
@@ -80,14 +81,14 @@ <h1 class="title-font sm:text-4xl text-3xl mb-4 font-extrabold">Keep your link s
...
 				class="container px-5 py-24 mx-auto flex md:items-center lg:items-start md:flex-row md:flex-no-wrap flex-wrap flex-col">
-				
-				
+
+
 			</div>
 			<div class="bg-gray-200">
 				<div class="container mx-auto py-4 px-5 flex flex-wrap flex-col sm:flex-row">
-					<p class="text-gray-500 text-sm text-center sm:text-left">© 2020 tailblocks —
+					<p class="text-gray-500 text-sm text-center sm:text-left">© 2020
```
Comment (micaiah-effiong) [inline public/index.html]: This tag is not necessary

**Review by micaiah-effiong — CHANGES_REQUESTED** [review]

**Review by Emilsone — COMMENTED** [review]
Made some changes on the footer

**Review by micaiah-effiong — APPROVED** [review]
Nice

**Pull request merged**